client/modules/product/components/ProductForm.js.jsx

```diff
@@ -48,6 +48,14 @@ function ProductForm({
               rows={3}
               placeholder="This is where the content goes..."
             />
+            <NumberInput
```

Comment:
when creating a new product, you will get an error as the page loads, "Failed prop type: The prop `value` is marked as required in `NumberInput`, but its value is `undefined`." could you try to figure out why this is and where to set a default value for the field (0)? it should be very similar to where 'title' and 'description' get their defaults (the empty string '').

Reply:
Definitely! Fixing it now.